src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java

```diff
@@ -79,10 +106,17 @@ public void theResponseStatusCodeShouldBeOnPatchAccountRequest(final String stat
 
     @When("I execute the delete Account request on {string} endpoint")
     public void iExecuteTheDeleteAccountRequestOnEndpoint(final String endpoint) {
+        requestBuilder
+                .addEndpoint("/Account/{accountID}")
+                .addPathParams("accountID", responseObject.getId().toString())
+                .addMethod(ApiMethod.DELETE)
+                .build();
+        apiResponse = ApiManager.execute(requestBuilder.build());
     }
 
     @Then("The response status code should be {string} on delete Account request")
     public void theResponseStatusCodeShouldBeOnDeleteAccountRequest(final String statusCode) {
+        Assert.assertEquals(apiResponse.getStatusCode(), 204);
```

Comment:
parameter not used. It should be used or deleted.

Reply:
Thanks, in the next PR I will fix it